dev/NavigationView/NavigationView.cpp

```diff
@@ -2161,8 +2161,12 @@ void NavigationView::OnNavigationViewItemTapped(const winrt::IInspectable& sende
 
 void NavigationView::OnNavigationViewItemKeyUp(const winrt::IInspectable& sender, const winrt::KeyRoutedEventArgs& args)
 {
-    // Because ListViewItem eats the events, we only get these keys on KeyUp.
-    if (args.OriginalKey() == winrt::VirtualKey::GamepadA)
+    // Because ListViewItem eats the events, we only get these keys on KeyUp AND
```

Comment:
This first line can now be deleted, it no longer applies after the repeater refactor

Reply:
Good point, removed.